src/java/org/apache/cassandra/index/sai/analyzer/LuceneAnalyzer.java

```diff
@@ -47,7 +47,7 @@
 
 public class LuceneAnalyzer extends AbstractAnalyzer
 {
-    public static final String JSON_ANALYZER = "json_analyzer";
+    public static final String JSON_INDEX_ANALYZER = "json_index_analyzer";
```

Comment:
One more thing - why is it called **json**_index_analyzer and not just index_analyzer?